src/opossum_lib/input_formats/owasp_dependency_scan/services/convert_to_opossum.py

```diff
@@ -57,6 +59,23 @@ def _extract_resources(
     return resources
 
 
+def _get_files_with_children(owasp_model: OWASPDependencyReportModel) -> list[str]:
+    files_with_children = []
+    for dependency in owasp_model.dependencies:
+        if dependency.is_virtual:
+            str_path = str(dependency.file_path) + "/"
```

Comment:
```suggestion
            str_path = dependency.file_path.as_posix() + "/"
```
is probably safer due to windows....